src/uv-common.c

```diff
@@ -113,6 +113,8 @@ const char* uv_err_name(int err) {
   switch (err) {
     UV_ERRNO_MAP(UV_ERR_NAME_GEN)
     default:
+      /* @20170517-sanggyu: To figure out what error number is missing */
+      printf("uv_err_name for (%d) is missing. Add err name string\n", err);
```

Comment:
@LaszloLango 
1. Both of `fprintf(stderr,...)` and `vfprintf(stderr, ...)` works for current tizenrt.
2. I changed to `TLOG`.
3. Here is binary size on Linux x86_64: (There is no binary size change in release mode.)

| build mode | before | after |
| --- | --- | --- |
| debug | 541584 | 544520 |
| release | 134452 | 134452 |